data/2022-results.json

```diff
@@ -62,5 +62,13 @@
         "count": 0
       }
     ]
+  },
+  "commute-distance-distribution": {
+    "5": 8,
+    "10": 40,
+    "15": 19,
+    "20": 48,
+    "25": 31,
+    "30": 68
```

Comment:
This is a minor thing at this point (since we are dealing with dummy data), but I thought is worth pointing out:
The values we are plotting on the bar charts are in percentages, so all the values plotted here should sum up to 100...